### PR TITLE
MDCT-296: Functionality and Seeding for FMAP and ACS Tables

### PR DESCRIPTION
**Pull request by karla-vm**
# Description

Closes [MDCT-296](https://qmacbis.atlassian.net/browse/MDCT-296). The `/state` endpoint should return FMAP and ACS data from the DB to load and populate Section 2 (Part 2) and Section 5 (Table 3) of CARTS reports.

## How to test

* Run locally `./dev local`
* Sign in to CARTS
* Open up a state report and navigate to **Section 2**
* Ensure the table in **Part 2: Number of Uninsured Children in Your State** is populated with either data or 'Not Available' (there should be no empty rows)
* Navigate to **Section 5**
* Ensure the _eFMAP_ row in **Table 3: Federal and State Shares** is populated with either data or 'Not Available' (there should be no empty rows)

Enjoy! 🪄 

## Dependencies

N/A

# Get it done

Cool things.

## Code authors checklist

- [x] I have performed a self-review of my code
- [ ] I have added thorough tests. [Testing Doc](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)
- [ ] Necessary analytics were added, or no new analytics were required
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned the PR to myself

## Reviewers Checklist (Two different people)

- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

## Assignee

- [ ] I have closed the PR after the review and necessary changes (squashing preferred)

### CODE REVIEW

**Review by gmrabian — DISMISSED** [review]
LGTM!

File: services/app-api/handlers/state/tests/get.test.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          fmapSet: [{}],
+          acsSet: [{}],
+        },
+      ],
+    }),
+  },
+}));
+
+jest.mock("../../../libs/authorization", () => ({
+  __esModule: true,
```
Comment (gmrabian) [inline services/app-api/handlers/state/tests/get.test.ts:27]: technically don't need `__esModule` here or on convertToDynamoExpressionVars. Only need it where you're mocking a default export, like with dynamodb-lib above :)

File: services/app-api/handlers/state/tests/get.test.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }),
+  },
+}));
+
+jest.mock("../../../libs/authorization", () => ({
+  isAuthorized: jest.fn().mockReturnValue(true),
+}));
+
+jest.mock("../../dynamoUtils/convertToDynamoExpressionVars", () => ({
+  __esModule: true,
```
Comment (gmrabian) [inline services/app-api/handlers/state/tests/get.test.ts:31]: can delete this one too :)

**Review by gmrabian — DISMISSED** [review]
Lovely! Just need codeclimate to play nice

**Comment by karla-vm** [comment]
> Lovely! Just need codeclimate to play nice

If only 😭

**Review by BearHanded — CHANGES_REQUESTED** [review]
Can we make the acs & fmap data their own tables? It might make updating that info less complicated in the long run when in X years a team is asked to update or add to the data set.

**Review by BearHanded — APPROVED** [review]

**Review by gmrabian — APPROVED** [review]
lgtm!

**Pull request merged**